packages/compass-components/src/components/accordion.tsx

```diff
@@ -12,7 +12,9 @@ const buttonStyles = css({
   fontWeight: 'bold',
   fontSize: defaultFontSize,
   display: 'flex',
-  alignItems: 'center',
+  alignItems: 'flex-start',
```

Comment:
- realign the chevron to the top, make sure the accordion looks right when the text breaks to multiple lines